log/devlog/devlog.go

```diff
@@ -0,0 +1,127 @@
+// Package devlog provides a plain-text logger suitable for reading from the
+// console.
+//
+// For production, the team prefers JSON logging, but plain-text can be useful
+// for local development and tests.
+//
+// So while this package shouldn't be used for production, it can be enabled
+// for your development use by setting TIDEPOOL_USE_DEVLOG=1 as an environment
+// variable. It will also be enabled if POD_NAMESPACE=default (this is set via
+// the development repo when using it's kind k8s cluster).
+
+package devlog
+
+import (
+	"fmt"
+	"io"
+	"log"
+	"sort"
+	"strings"
+	"time"
+
+	"github.com/tidepool-org/platform/errors"
+	plog "github.com/tidepool-org/platform/log"
```

Comment:
`plog` to `platformLog`? Or, as we've done in other code, use `stdLog` for the standard library log. (Similar to `stdError`.) 

IMHO, except in certain, very common cases (i.e. "ID"), abbreviations tend to lead to confusion. What makes sense to one person as an abbreviation may mean something else to another.